webhook/handlers/patient_first_workflow.js

```diff
@@ -0,0 +1,71 @@
+const db = require('../../models')
+
+class PatientFirstController {
+
+  async bookConsulation(agent, id){
+    let fulfillmentText = "";
+    let user = await db.userWA.findOne({where: {wa_phone_number: id}});
+    let patients = await db.patientWA.findAll({where: { user_id : user.id }});
+        if(patients.length == 0){
+            fulfillmentText += ("We don't have any patients registered with us. What would you like to do?\
```

Comment:
`\` is redundant and should be omitted, since you can enclose each line in `quotes` if you're using parenthesis.
I prefer using `+` like in: https://github.com/kj7kunal/E-Med/blob/develop-patient-first-workflow/webhook/handlers/user_registration_workflow.js#L135-L142
But `+` is only required when concatenating a variable string. I will also remove `+` later on.